### PR TITLE
Bump version, minor fixes, and rerender

### PR DESCRIPTION
**Pull request by dalcinl**

### CODE REVIEW

**Comment by dalcinl** [comment]
@minrk This one is ready merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,9 +31,9 @@ test:
         - tests/helloworld.cxx
         - tests/helloworld.f
         - tests/helloworld.f90
-    commands:
-        - conda inspect objects openmpi  # [osx]
-        - conda inspect linkages openmpi  # [not win]
+#   commands:
+#       - conda inspect objects openmpi  # [osx]
```
Comment (minrk) [inline recipe/meta.yaml]: Did you verify that these fail? I believe it's a specific bug in mpich that causes O_NONBLOCK problems.

Comment (dalcinl) [reply recipe/meta.yaml]: No, sorry, you are right. Do you think I should revert these changes?

Comment (dalcinl) [reply recipe/meta.yaml]: Never mind. I'm going to strip this commit.

**Comment by dalcinl** [comment]
@minrk I stripped the commit removing the `conda inspect` tests. Let's wait for the builds, if everything is fine, I'll merge.

**Comment by dalcinl** [comment]
@minrk I will delay the merge of this PR until I can give it a check to the macOS builds in my laptop. The output of `conda inspect objects openmpi` shows rpath entries that seems unnecessary, as `@loader_path`  entries should be enough to resolve libraries. Thus, I would like to play with the `build.sh` script, maybe we can get better binaries that do not need rewrites for relocation at package install time.

**Pull request merged**